CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## 0.7.1 - 2018-05-09
-### Fixed
-[Bug fixed](https://github.com/Microsoft/vscode-cosmosdb/issues/623)
+## 0.7.1 - 2018-05-10
+
+### [Fixed](https://github.com/Microsoft/vscode-cosmosdb/issues?q=is%3Aissue+milestone%3A%220.7.1%22+is%3Aclosed+label%3Abug)
+
+- Fixed error "Unexpected Experience Value" when retrieving database accounts
```

Comment:
Thanks for making the milestone! 

I followed the convention here: https://github.com/Microsoft/vscode-cosmosdb/pull/626/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR64

Reply:
Lol you made that "convention" yourself: https://github.com/Microsoft/vscode-cosmosdb/pull/439

In any case - I think it's important to have the bug listed by itself rather than making the user click a link just to see what changed (especially for a bug-fix release where this is the only change)